api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
 // +build !ignore_autogenerated
```

Comment:
I think no need to change this file. thanks.

Reply:
it is auto generated by vscode. May be it doesn't matter like [this](https://github.com/kubernetes/client-go/blob/master/scale/scheme/zz_generated.deepcopy.go#L1).